packages/web/src/components/OpenSourceDatasets/DatasetTable.module.css

```diff
@@ -0,0 +1,57 @@
+.table {
+  padding-bottom: calc(2*var(--margin));
+}
+
+.table-header > div {
+  background-color: var(--secondary-dark);
+  background: var(--secondary-dark);
+  border-bottom: 1px solid var(--borders-light-grey);
+}
+
+.table-header span {
+  color: var(--secondary-text-color);
+  font-family: var(--font-family);
+  font-size: 12px;
+  font-weight: 400;
+}
+
+.table-header > div:hover, .table-header span:hover {
+  background-color: var(--secondary-dark);
+  background: var(--secondary-dark);
+  color: var(--secondary-text-color);
+}
+
+.table-content > div {
+  color: var(--secondary-text-color);
+  background-color: var(--secondary-dark);
+  background: var(--secondary-dark) !important;
+  border-bottom: 1px solid var(--borders-light-grey);
+  font-family: var(--font-family);
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.table-content > div:last-of-type {
+  border-bottom: none;
+}
+
+/* .table-content div {
```

Comment:
Lines in table should wrap once before using overflow ellipses
TO DO:  Convert to ticket